subsys/bluetooth/host/keys.c

```diff
@@ -37,6 +39,38 @@ static struct bt_keys key_pool[CONFIG_BT_MAX_PAIRED];
 #if IS_ENABLED(CONFIG_BT_KEYS_OVERWRITE_OLDEST)
 static uint32_t aging_counter_val;
 static struct bt_keys *last_keys_updated;
+
+struct key_data {
+	bool in_use;
+	uint8_t id;
+};
+
+static void find_key_in_use(struct bt_conn *conn, void *data)
+{
+	struct key_data *kdata = (struct key_data *)data;
+	struct bt_keys *key;
+
+	if (conn->state == BT_CONN_CONNECTED) {
+		key = bt_keys_find_addr(conn->id, bt_conn_get_dst(conn));
+		if (key == NULL) {
+			return;
+		}
```

Comment:
really minor, but to keep consistent with other Bluetooth code: add an empty line after } unless the next line is also }

Reply:
Done